### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by gpavlov2016** [onflow/developer-portal#735]
**Source**
Algolia crawler "ignored" links list, manually filtered to exclude redirects and links that are not broken

**Task**
1. Determine if the link is broken in github md page (assume yes)
2. Figure out what the correct link should be

**Format**
- [broken link url]
[referer 1]
...
[referer n]

**List of broken links and referrers**

link list distributed down below to get fixed

**Comment by fotescodev** [comment]
Links taken:
https://developers.flow.com/fusd/transactions/
https://developers.flow.com/flow/fusd/index

https://developers.flow.com/fusd/providers/
https://developers.flow.com/flow/fusd/index

https://developers.flow.com/kitty-items/next-steps/
https://developers.flow.com/flow/index

https://developers.flow.com/flow-token/earn
https://developers.flow.com/flow/flow-token/index

https://developers.flow.com/flow-token/delivery
https://developers.flow.com/flow/flow-token/index

https://developers.flow.com/tools/flow-go-sdk/
https://developers.flow.com/tools/flow-go-sdk/index

https://developers.flow.com/tools/building-transactions
https://developers.flow.com/tools/flow-go-sdk/index

https://developers.flow.com/tools/unity-sdk/dev-wallet
https://developers.flow.com/tools/unity-sdk/samples/ui-usage

https://developers.flow.com/tools/fcl-js/configure-fcl
https://developers.flow.com/tools/fcl-js/reference/scripts

**Comment by srinjoyc** [comment]
Links taken:
- https://developers.flow.com/tools/unity-sdk/
https://developers.flow.com/tools/unity-sdk/index |  
https://developers.flow.com/tools/unity-sdk/guides/flow-control |  
https://developers.flow.com/tools/unity-sdk/samples/quickstart |  
https://developers.flow.com/tools/unity-sdk/samples/ui-usage |  
https://developers.flow.com/tools/unity-sdk/samples/flow-words-tutorial |  
https://developers.flow.com/tools/unity-sdk/api-reference |  
https://developers.flow.com/tools/unity-sdk/guides/dev-wallet
- https://developers.flow.com/flow/nft-marketplace/fusd/
https://developers.flow.com/flow/nft-marketplace/building-blocks
- https://developers.flow.com/flow/nft-marketplace/flow-token/
https://developers.flow.com/flow/nft-marketplace/building-blocks
- https://developers.flow.com/flow/nft-marketplace/fcl/
https://developers.flow.com/flow/nft-marketplace/building-blocks |  
https://developers.flow.com/flow/nft-marketplace/handling-accounts
- https://developers.flow.com/flow/nft-marketplace/access-api/
https://developers.flow.com/flow/nft-marketplace/building-blocks
- https://developers.flow.com/flow/nft-marketplace/cadence/
https://developers.flow.com/flow/nft-marketplace/building-blocks
- https://developers.flow.com/flow/nft-marketplace/flip-fest-winners/
https://developers.flow.com/flow/nft-marketplace/building-blocks
- https://developers.flow.com/flow/nft-marketplace/flow-go-sdk/
https://developers.flow.com/flow/nft-marketplace/building-blocks

**Comment by 10thfloor** [comment]
👋🏼 @srinjoyc @fotescodev 

I want to chime in here in the hopes of making this less painful for you all. 

Manually verifying links on pages turned up by Algolia is a _waste of time_. 

The solution to your pain lies in **finishing the link checker PR**: https://github.com/onflow/developer-portal/pull/717
It's 99% code-complete, with one case left to validate: relative links to content outside of the root `docs` folder.  All other cases are covered. 

Once that PR is merged, you won't need Algolia. Broken links will turn up in the check runs of all repos with docs content. The link checker _clicks every link on a page and captures the response_, and lists those links in the check-run, Automating the work you're doing here, offloading the updates to the teams responsible.

Check runs can be set to block merges, and enforce that teams _fix their links themselves_ in perpetuity, guaranteeing that links get fixed. 

**The rules for updating links are simple:** All links to content in the same collection (repo) should be relative*. No knowledge of the docs site URL structure is needed.
 
It has been known _for a long time_ that manually updating a bunch of links would be necessary. Making links work in GitHub _and_ on the portal was part of the original design. We've had to do something similar for the previous 2 docs site iterations. 

The rollout for updating the entire corpus of docs was planned with Tanooki as follows, 
and I think this is still the best way forward: 

1) Finish link checker
2) Announce to teams that links will need to be updated. (rules for updating are documented in Notion, and in this comment)
2) Enforce check runs block merges while links are broken.
3) Support teams fixing their links
4) Never worry about broken links again.

---
*Except for the `onflow/flow` repo, where links can only be relative to their _containing folder_.

**Comment by bthaile** [comment]
@10thfloor Thanks very much for the write up 🙏 
Question as far as you know can #717 (https://github.com/onflow/developer-portal/pull/717) be merged up and in a separate effort fix the content checker to handle `relative links to content outside of the root docs folder.` Since it'll take a bit to ramp up on the content checker. 
Is the documentation on how to configure and literately dev on the content checker?

**Comment by bthaile** [comment]
- https://developers.flow.com/access-api
https://developers.flow.com/nodes/access-api-rate-limits |  
https://developers.flow.com/flow/faq/developers
- https://developers.flow.com/flow/core-contracts/undefined
https://developers.flow.com/flow/core-contracts/nft-storefront
- https://developers.flow.com/flow/locked-tokens/
https://developers.flow.com/flow/core-contracts/staking-contract-reference
- https://developers.flow.com/to:team@coinlist.co
https://developers.flow.com/nodes/staking/custody-providers
- https://developers.flow.com/to:support@finoa.com
https://developers.flow.com/nodes/staking/custody-providers
- https://developers.flow.com/concepts/transaction-lifecycle
https://developers.flow.com/nodes/access-api
- https://developers.flow.com/access-api-rate-limits
https://developers.flow.com/nodes/access-api
- https://developers.flow.com/flow-mainnet
https://developers.flow.com/nodes/index |  
https://developers.flow.com/nodes/access-api-rate-limits |  
https://developers.flow.com/nodes/access-api
- https://developers.flow.com/nodes/flow-mainnet
https://developers.flow.com/sitemap.xml

**Comment by gpavlov2016** [comment]
Approved and merged the first 3 PRs from the list above.
The change in the last PR doesn't work - it still shows 404

**Comment by bthaile** [comment]
All mailto links will show 404. It's a known issue tracked by this ticket.
https://github.com/onflow/developer-portal/issues/666

**Comment by gpavlov2016** [comment]
Merged the last PR

**Comment by huyndo** [comment]
- https://developers.flow.com/to:support@finoa.io
https://developers.flow.com/flow/flow-token/delivery |  
https://developers.flow.com/flow/flow-token/available-wallets
- https://developers.flow.com/to:hello@onflow.org
https://developers.flow.com/flow/flow-token/backers
- https://developers.flow.com/to:flow@finoa.io
https://developers.flow.com/flow/flow-token/delivery |  
https://developers.flow.com/flow/flow-token/available-wallets
- https://developers.flow.com/tools/flow-go-sdk/creating-accounts
https://developers.flow.com/flow/faq/developers

**Comment by bthaile** [comment]
@huyndo  Can you look at these broken links
 - (not linkable, missing nav) - https://developers.flow.com/concepts/transaction-lifecycle
https://developers.flow.com/nodes/access-api
 - https://developers.flow.com/access-api-rate-limits
https://developers.flow.com/nodes/access-api
 - https://developers.flow.com/flow-mainnet
https://developers.flow.com/nodes/index |  
https://developers.flow.com/nodes/access-api-rate-limits |  
https://developers.flow.com/nodes/access-api
 - https://developers.flow.com/nodes/flow-mainnet
https://developers.flow.com/sitemap.xml

**Comment by gpavlov2016** [comment]
Update: using a different service (than algolia), we were able to scan external links and the updated count of broken links is now 29:

```
Result | URL |  
-- | -- | --
404 Not found | https://github.com/onflow/developer-portal/blob/main/app/data/pages/home.ts<a href>OutboundLinked from: https://developers.flow.com/ and 2 more |  
404 Not found | https://developers.flow.com/images/content/reference.svgCSS url()Linked from: https://developers.flow.com/build/_assets/main-7FCEA254.css |  
404 Not found | https://github.com/onflow/flow-go-sdk/blob/master/examples/rest_grpc_clients/main.go<a href>OutboundLinked from: https://developers.flow.com/tools/flow-go-sdk and 3 more |  
404 Not found | https://pbs.twimg.com/profile_banners/1458578264306462723/1638906249/1500x500<img src>OutboundLinked from: https://developers.flow.com/community |  
404 Not found | https://pbs.twimg.com/profile_images/1422256068554641413/iR0aoxSO_400x400.jpg<img src>OutboundLinked from: https://developers.flow.com/community |  
404 Not found | https://developers.flow.com/build/_assets/tool-port-M3XDCSUS.svg<img src>Linked from: https://developers.flow.com/community |  
404 Not found | https://developers.flow.com/tools/building-transactions<a href>Linked from: https://developers.flow.com/tools/flow-go-sdk and 3 more |  
404 Not found | https://github.com/onflow/cadence/tree/master/languageserver<a href>OutboundLinked from: https://developers.flow.com/tools/vscode-extension and 3 more |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/web/src/hooks/use-current-user.hook.js<a href>OutboundLinked from: https://developers.flow.com/tools/fcl-js/reference/api and 1 more |  
404 Not found | https://docs.onflow.org/access-api/<a href>OutboundRedirectedLinked from: https://developers.flow.com/tools/fcl-js/reference/sdk-guidelines and 3 more |  
404 Not found | https://github.com/onflow/flow-port/blob/staging/src/pages/transaction-status.js<a href>OutboundLinked from: https://developers.flow.com/tools/fcl-js/reference/api and 1 more |  
404 Not found | https://developers.flow.com/tools/fcl-js/configure-fcl<a href>Linked from: https://developers.flow.com/tools/fcl-js/reference/scripts and 1 more |  
Host not found | https://www.versus-flow.art/<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/user-accounts-and-wallets |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/api/src/workers/sale-offer-handler.ts<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/web/src/flow/script.get-market-item.js<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/api/src/services/market.ts<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure |  
404 Not found | https://developers.flow.com/port-login.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup |  
404 Not found | https://developers.flow.com/select-wallet.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup |  
404 Not found | https://developers.flow.com/port-dashboard.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup |  
404 Not found | https://developers.flow.com/flowscan-fees.png<img src>Linked from: https://developers.flow.com/learn/concepts/variable-transaction-fees and 1 more |  
404 Not found | https://developers.flow.com/flow-token/delivery<a href>Linked from: https://developers.flow.com/flow/flow-token |  
404 Not found | https://developers.flow.com/flow-token/earn<a href>Linked from: https://developers.flow.com/flow/flow-token |  
404 Not found | https://github.com/onflow/open-source-template<a href>OutboundLinked from: https://developers.flow.com/cadence/style-guide/project-development-tips |  
404 Not found | http://developers.flow.com/flow/access-apiSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/access-api | Details
Connect error | http://localhost:3001/<a href>OutboundLinked from: https://developers.flow.com/learn/kitty-items/start and 1 more |  
Connect error | http://localhost:3001/admin/mint/<a href>OutboundLinked from: https://developers.flow.com/learn/kitty-items/start and 1 more |  
404 Not found | https://developers.flow.com/flow-mainnet<a href>Linked from: https://developers.flow.com/nodes |  
404 Not found | https://github.com/onflow/flow-core-contracts/blob/master/transactions/lockedTokens/admin/create_admin_collection.cdc<a href>OutboundLinked from: https://developers.flow.com/flow/core-contracts/locked-tokens |  
404 Not found | https://flow-docs-8kbrufdlq-onflow.vercel.app/nodes/node-operation/node-bootstrap/<a href>OutboundNoindexLinked from: https://developers.flow.com/nodes/node-operation/node-setup

Result	URL	
404 Not found	
https://github.com/onflow/developer-portal/blob/main/app/data/pages/home.ts<a href>Outbound
Linked from: https://developers.flow.com/ and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://developers.flow.com/images/content/reference.svgCSS url()
Linked from: https://developers.flow.com/build/_assets/main-7FCEA254.css
404 Not found	
https://github.com/onflow/flow-go-sdk/blob/master/examples/rest_grpc_clients/main.go<a href>Outbound
Linked from: https://developers.flow.com/tools/flow-go-sdk and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://pbs.twimg.com/profile_banners/1458578264306462723/1638906249/1500x500<img src>Outbound
Linked from: https://developers.flow.com/community
404 Not found	
https://pbs.twimg.com/profile_images/1422256068554641413/iR0aoxSO_400x400.jpg<img src>Outbound
Linked from: https://developers.flow.com/community
404 Not found	
https://developers.flow.com/build/_assets/tool-port-M3XDCSUS.svg<img src>
Linked from: https://developers.flow.com/community
404 Not found	
https://developers.flow.com/tools/building-transactions<a href>
Linked from: https://developers.flow.com/tools/flow-go-sdk and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://github.com/onflow/cadence/tree/master/languageserver<a href>Outbound
Linked from: https://developers.flow.com/tools/vscode-extension and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://github.com/onflow/kitty-items/blob/master/web/src/hooks/use-current-user.hook.js<a href>Outbound
Linked from: https://developers.flow.com/tools/fcl-js/reference/api and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://docs.onflow.org/access-api/<a href>OutboundRedirected
Linked from: https://developers.flow.com/tools/fcl-js/reference/sdk-guidelines and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://github.com/onflow/flow-port/blob/staging/src/pages/transaction-status.js<a href>Outbound
Linked from: https://developers.flow.com/tools/fcl-js/reference/api and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://developers.flow.com/tools/fcl-js/configure-fcl<a href>
Linked from: https://developers.flow.com/tools/fcl-js/reference/scripts and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
Host not found	
https://www.versus-flow.art/<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/user-accounts-and-wallets
404 Not found	
https://github.com/onflow/kitty-items/blob/master/api/src/workers/sale-offer-handler.ts<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure
404 Not found	
https://github.com/onflow/kitty-items/blob/master/web/src/flow/script.get-market-item.js<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure
404 Not found	
https://github.com/onflow/kitty-items/blob/master/api/src/services/market.ts<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure
404 Not found	
https://developers.flow.com/port-login.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup
404 Not found	
https://developers.flow.com/select-wallet.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup
404 Not found	
https://developers.flow.com/port-dashboard.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup
404 Not found	
https://developers.flow.com/flowscan-fees.png<img src>
Linked from: https://developers.flow.com/learn/concepts/variable-transaction-fees and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://developers.flow.com/flow-token/delivery<a href>
Linked from: https://developers.flow.com/flow/flow-token
404 Not found	
https://developers.flow.com/flow-token/earn<a href>
Linked from: https://developers.flow.com/flow/flow-token
404 Not found	
https://github.com/onflow/open-source-template<a href>Outbound
Linked from: https://developers.flow.com/cadence/style-guide/project-development-tips
404 Not found	
http://developers.flow.com/flow/access-apiSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/access-api
Connect error	
http://localhost:3001/<a href>Outbound
Linked from: https://developers.flow.com/learn/kitty-items/start and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
Connect error	
http://localhost:3001/admin/mint/<a href>Outbound
Linked from: https://developers.flow.com/learn/kitty-items/start and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=50#)
404 Not found	
https://developers.flow.com/flow-mainnet<a href>
Linked from: https://developers.flow.com/nodes
404 Not found	
https://github.com/onflow/flow-core-contracts/blob/master/transactions/lockedTokens/admin/create_admin_collection.cdc<a href>Outbound
Linked from: https://developers.flow.com/flow/core-contracts/locked-tokens
404 Not found	
https://flow-docs-8kbrufdlq-onflow.vercel.app/nodes/node-operation/node-bootstrap/<a href>OutboundNoindex
Linked from: https://developers.flow.com/nodes/node-operation/node-setup
```
<img width="1483" alt="image" src="https://user-images.githubusercontent.com/16441017/204021389-758e6069-98ec-468c-a558-5d95af1ae810.png">

**Comment by gpavlov2016** [comment]
The number of broken links has skyrocketed by 211% to 117 links. Updated list:
```
404 Not found | https://github.com/onflow/developer-portal/blob/main/app/data/pages/home.ts<a href>OutboundLinked from: https://developers.flow.com/ and 2 more |  
404 Not found | https://developers.flow.com/network/flow-mainnet<a href>Linked from: https://developers.flow.com/ and 10+ more |  
404 Not found | https://developers.flow.com/images/content/reference.svgCSS url()Linked from: https://developers.flow.com/build/_assets/main-MI5KYKUU.css |  
500 Internal server error | https://developers.flow.com/assets/cadence/tutorial/deploybox.png<img src>Linked from: https://developers.flow.com/cadence/tutorial/02-hello-world and 4 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/flow-dapp-anatomy.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/flow-dapp-anatomy and 1 more |  
500 Internal server error | https://developers.flow.com/flow/fusd<a href>Linked from: https://developers.flow.com/flow/fungible-tokens and 9 more |  
500 Internal server error | https://developers.flow.com/assets/learn/concepts/transaction-anatomy.png<img src>Linked from: https://developers.flow.com/learn/concepts/accounts-and-keys and 6 more |  
500 Internal server error | https://developers.flow.com/assets/flow/flow-ecosystem.png<img src>Linked from: https://developers.flow.com/flow/fungible-tokens |  
500 Internal server error | https://developers.flow.com/assets/flow/token-allocation.png<img src>Linked from: https://developers.flow.com/flow/fungible-tokens |  
500 Internal server error | https://developers.flow.com/assets/flow/token-distribution.png<img src>Linked from: https://developers.flow.com/flow/fungible-tokens |  
500 Internal server error | https://developers.flow.com/assets/tools/flow-go-sdk/sdk-banner.svg<img src>Linked from: https://developers.flow.com/tools/flow-go-sdk and 3 more |  
404 Not found | https://github.com/onflow/flow-go-sdk/blob/master/examples/rest_grpc_clients/main.go<a href>OutboundLinked from: https://developers.flow.com/tools/flow-go-sdk and 3 more |  
404 Not found | https://pbs.twimg.com/profile_banners/1458578264306462723/1638906249/1500x500<img src>OutboundLinked from: https://developers.flow.com/community |  
500 Internal server error | https://developers.flow.com/assets/tools/fcl-js/images/discovery.png<img src>Linked from: https://developers.flow.com/tools/fcl-js and 4 more |  
404 Not found | https://pbs.twimg.com/profile_images/1422256068554641413/iR0aoxSO_400x400.jpg<img src>OutboundLinked from: https://developers.flow.com/community |  
404 Not found | https://developers.flow.com/build/_assets/tool-port-M3XDCSUS.svg<img src>Linked from: https://developers.flow.com/community |  
404 Not found | https://developers.flow.com/mobile/design-patterns<a href>NewLinked from: https://developers.flow.com/mobile/index and 3 more |  
404 Not found | https://developers.flow.com/tools/building-transactions<a href>Linked from: https://developers.flow.com/tools/flow-go-sdk and 3 more |  
404 Not found | https://developers.flow.com/mobile/contract-upgrades<a href>NewLinked from: https://developers.flow.com/mobile/index and 3 more |  
404 Not found | https://github.com/onflow/cadence/tree/master/languageserver<a href>OutboundLinked from: https://developers.flow.com/tools/vscode-extension and 3 more |  
503 Service unavailable | https://kitty-items.onflow.org/<a href>OutboundRedirectedLinked from: https://developers.flow.com/learn/kitty-items and 2 more |  
500 Internal server error | https://developers.flow.com/assets/learn/kitty-items/landing-page.png<img src>Linked from: https://developers.flow.com/learn/kitty-items and 2 more |  
500 Internal server error | https://developers.flow.com/assets/learn/kitty-items/kitty-items-diagram.png<img src>Linked from: https://developers.flow.com/learn/kitty-items and 2 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image4.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image2.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image3.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image7.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image8.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image1.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image6.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/architectureImages/image5.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and 1 more |  
503 Service unavailable | http://kitty-items-flow-testnet.herokuapp.com/<a href>OutboundLinked from: https://developers.flow.com/tools/fcl-js/reference/api and 1 more |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/web/src/hooks/use-current-user.hook.js<a href>OutboundLinked from: https://developers.flow.com/tools/fcl-js/reference/api and 1 more |  
404 Not found | https://docs.onflow.org/access-api/<a href>OutboundRedirectedLinked from: https://developers.flow.com/tools/fcl-js/reference/api and 3 more |  
404 Not found | https://github.com/onflow/flow-port/blob/staging/src/pages/transaction-status.js<a href>OutboundLinked from: https://developers.flow.com/tools/fcl-js/reference/api and 1 more |  
404 Not found | https://developers.flow.com/tools/fcl-js/configure-fcl<a href>Linked from: https://developers.flow.com/tools/fcl-js/reference/scripts and 1 more |  
500 Internal server error | https://developers.flow.com/assets/tools/fcl-js/images/wc-discovery.png<img src>Linked from: https://developers.flow.com/tools/fcl-js/reference/wallet-connect |  
SSL handshake error | https://www.versus-flow.art/<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/user-accounts-and-wallets |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/api/src/workers/sale-offer-handler.ts<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/web/src/flow/script.get-market-item.js<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure |  
404 Not found | https://github.com/onflow/kitty-items/blob/master/api/src/services/market.ts<a href>OutboundLinked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/NFTimages/image1.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/NFTimages/image2.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/NFTimages/image3.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/NFTimages/image4.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/NFTimages/image5.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/NFTimages/image6.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles |  
404 Not found | https://developers.flow.com/port-login.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
404 Not found | https://developers.flow.com/select-wallet.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
404 Not found | https://developers.flow.com/port-dashboard.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/port-new-account.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/port-approve.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/port-event.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
500 Internal server error | https://developers.flow.com/assets/flow/dapp-development/port-send.png<img src>Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and 1 more |  
404 Not found | https://developers.flow.com/flowscan-fees.png<img src>Linked from: https://developers.flow.com/learn/concepts/variable-transaction-fees and 1 more |  
404 Not found | https://sandboxnet-faucet.onflow.org/<a href>OutboundNewLinked from: https://developers.flow.com/learn/concepts/accessing-sandboxnet |  
500 Internal server error | https://developers.flow.com/assets/learn/concepts/port-sealed-tx.png<img src>NewLinked from: https://developers.flow.com/learn/concepts/accessing-mainnet |  
404 Not found | https://developers.flow.com/flow-token/delivery<a href>Linked from: https://developers.flow.com/flow/flow-token and 2 more |  
404 Not found | https://developers.flow.com/flow-token/earn<a href>Linked from: https://developers.flow.com/flow/flow-token and 2 more |  
500 Internal server error | https://developers.flow.com/flow/nft-marketplace/building-blocks<a href>Linked from: https://developers.flow.com/flow/nft-marketplace and 6 more |  
500 Internal server error | https://developers.flow.com/flow/nft-marketplace/best-practices<a href>Linked from: https://developers.flow.com/flow/nft-marketplace and 6 more |  
500 Internal server error | https://developers.flow.com/flow/nft-marketplace/handling-accounts<a href>Linked from: https://developers.flow.com/flow/nft-marketplace and 6 more |  
500 Internal server error | https://developers.flow.com/flow/nft-marketplace/minting-nfts<a href>Linked from: https://developers.flow.com/flow/nft-marketplace and 6 more |  
500 Internal server error | https://developers.flow.com/flow/nft-marketplace/selling-nfts<a href>Linked from: https://developers.flow.com/flow/nft-marketplace and 6 more |  
500 Internal server error | http://developers.flow.com/flow/fusdSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/fusd and 2 more |  
500 Internal server error | https://developers.flow.com/flow/fusd/<a href>RedirectedLinked from: https://developers.flow.com/flow/fusd and 3 more |  
404 Not found | https://github.com/onflow/open-source-template<a href>OutboundLinked from: https://developers.flow.com/cadence/style-guide/project-development-tips and 2 more |  
404 Not found | http://developers.flow.com/mobile/design-patternsSocial meta tagRedirectedNewLinked from: https://developers.flow.com/mobile/design-patterns |  
404 Not found | http://developers.flow.com/mobile/contract-upgradesSocial meta tagRedirectedNewLinked from: https://developers.flow.com/mobile/contract-upgrades |  
404 Not found | http://developers.flow.com/flow/access-apiSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/access-api |  
Connect error | http://localhost:3001/<a href>OutboundLinked from: https://developers.flow.com/learn/kitty-items/start and 2 more |  
500 Internal server error | https://developers.flow.com/assets/learn/kitty-items/welcome-ui.png<img src>Linked from: https://developers.flow.com/learn/kitty-items/start and 1 more |  
500 Internal server error | https://developers.flow.com/assets/learn/kitty-items/admin-login.png<img src>Linked from: https://developers.flow.com/learn/kitty-items/start and 1 more |  
Connect error | http://localhost:3001/admin/mint/<a href>OutboundLinked from: https://developers.flow.com/learn/kitty-items/start and 3 more |  
500 Internal server error | https://developers.flow.com/assets/learn/kitty-items/admin-ui.png<img src>Linked from: https://developers.flow.com/learn/kitty-items/start and 3 more |  
500 Internal server error | https://developers.flow.com/assets/learn/kitty-items/shades-nft.png<img src>Linked from: https://developers.flow.com/learn/kitty-items/modify and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/node-operation/flow-architecture.png<img src>Linked from: https://developers.flow.com/nodes/node-operation/node-setup and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/staking/epoch-service-event-diagram.png<img src>Linked from: https://developers.flow.com/nodes/staking/epoch-scripts-events and 3 more |  
404 Not found | https://github.com/onflow/flow-core-contracts/blob/master/transactions/lockedTokens/admin/create_admin_collection.cdc<a href>OutboundLinked from: https://developers.flow.com/flow/core-contracts/locked-tokens and 1 more |  
404 Not found | https://github.com/onflow/flow/blob/master/docs/content/access-api.md<a href>OutboundLinked from: https://developers.flow.com/nodes/node-operation/observer-node |  
500 Internal server error | https://developers.flow.com/assets/nodes/node-operation/observer.png<img src>Linked from: https://developers.flow.com/nodes/node-operation/observer-node |  
404 Not found | https://developers.flow.com/concepts/transaction-lifecycle<a href>Linked from: https://developers.flow.com/nodes/access-api |  
500 Internal server error | https://developers.flow.com/assets/cadence/language/type-hierarchy.png<img src>Linked from: https://developers.flow.com/cadence/language/type-hierarchy |  
500 Internal server error | https://developers.flow.com/assets/flow/flow-token/dapper-web.png<img src>Linked from: https://developers.flow.com/flow/flow-token/available-wallets |  
500 Internal server error | https://developers.flow.com/assets/flow/flow-token/dapper-login.png<img src>Linked from: https://developers.flow.com/flow/flow-token/available-wallets |  
404 Not found | http://developers.flow.com/tools/fcl-js/configure-fclSocial meta tagRedirectedLinked from: https://developers.flow.com/tools/fcl-js/configure-fcl |  
500 Internal server error | https://developers.flow.com/fusd/<a href>RedirectedLinked from: https://developers.flow.com/flow/fusd/providers |  
404 Not found | https://support.ledger.com/hc/en-us/articles/360006395553<a href>OutboundNoindexLinked from: https://developers.flow.com/nodes/flow-port and 3 more |  
500 Internal server error | http://developers.flow.com/flow/nft-marketplace/building-blocksSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/nft-marketplace/building-blocks |  
500 Internal server error | http://developers.flow.com/flow/nft-marketplace/best-practicesSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/nft-marketplace/best-practices |  
500 Internal server error | http://developers.flow.com/flow/nft-marketplace/handling-accountsSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/nft-marketplace/handling-accounts |  
500 Internal server error | http://developers.flow.com/flow/nft-marketplace/minting-nftsSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/nft-marketplace/minting-nfts |  
500 Internal server error | http://developers.flow.com/flow/nft-marketplace/selling-nftsSocial meta tagRedirectedLinked from: https://developers.flow.com/flow/nft-marketplace/selling-nfts |  
404 Not found | https://access-testnet.onflow.org/<a href>OutboundLinked from: https://developers.flow.com/flow/faq/operators |  
404 Not found | https://access-sandboxnet.onflow.org/<a href>OutboundLinked from: https://developers.flow.com/flow/faq/operators |  
404 Not found | https://access.onflow.org/<a href>OutboundLinked from: https://developers.flow.com/flow/faq/operators |  
500 Internal server error | https://developers.flow.com/nodes/(https://developers.flow.com/nodes/access-api-rate-limits)<a href>NewLinked from: https://developers.flow.com/nodes/archive-api-rate-limits |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-0-00.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
404 Not found | https://docs.onflow.org/concepts/nodes/node-operation/quickstart/<a href>OutboundRedirectedLinked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-0-02.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more

Result	URL	
404 Not found	
https://github.com/onflow/developer-portal/blob/main/app/data/pages/home.ts<a href>Outbound
Linked from: https://developers.flow.com/ and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/network/flow-mainnet<a href>
Linked from: https://developers.flow.com/ and [10+ more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/images/content/reference.svgCSS url()
Linked from: https://developers.flow.com/build/_assets/main-MI5KYKUU.css
500 Internal server error	
https://developers.flow.com/assets/cadence/tutorial/deploybox.png<img src>
Linked from: https://developers.flow.com/cadence/tutorial/02-hello-world and [4 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/flow-dapp-anatomy.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/flow-dapp-anatomy and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/fusd<a href>
Linked from: https://developers.flow.com/flow/fungible-tokens and [9 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/concepts/transaction-anatomy.png<img src>
Linked from: https://developers.flow.com/learn/concepts/accounts-and-keys and [6 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/flow-ecosystem.png<img src>
Linked from: https://developers.flow.com/flow/fungible-tokens
500 Internal server error	
https://developers.flow.com/assets/flow/token-allocation.png<img src>
Linked from: https://developers.flow.com/flow/fungible-tokens
500 Internal server error	
https://developers.flow.com/assets/flow/token-distribution.png<img src>
Linked from: https://developers.flow.com/flow/fungible-tokens
500 Internal server error	
https://developers.flow.com/assets/tools/flow-go-sdk/sdk-banner.svg<img src>
Linked from: https://developers.flow.com/tools/flow-go-sdk and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/flow-go-sdk/blob/master/examples/rest_grpc_clients/main.go<a href>Outbound
Linked from: https://developers.flow.com/tools/flow-go-sdk and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://pbs.twimg.com/profile_banners/1458578264306462723/1638906249/1500x500<img src>Outbound
Linked from: https://developers.flow.com/community
500 Internal server error	
https://developers.flow.com/assets/tools/fcl-js/images/discovery.png<img src>
Linked from: https://developers.flow.com/tools/fcl-js and [4 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://pbs.twimg.com/profile_images/1422256068554641413/iR0aoxSO_400x400.jpg<img src>Outbound
Linked from: https://developers.flow.com/community
404 Not found	
https://developers.flow.com/build/_assets/tool-port-M3XDCSUS.svg<img src>
Linked from: https://developers.flow.com/community
404 Not found	
https://developers.flow.com/mobile/design-patterns<a href>New
Linked from: https://developers.flow.com/mobile/index and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/tools/building-transactions<a href>
Linked from: https://developers.flow.com/tools/flow-go-sdk and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/mobile/contract-upgrades<a href>New
Linked from: https://developers.flow.com/mobile/index and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/cadence/tree/master/languageserver<a href>Outbound
Linked from: https://developers.flow.com/tools/vscode-extension and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
503 Service unavailable	
https://kitty-items.onflow.org/<a href>OutboundRedirected
Linked from: https://developers.flow.com/learn/kitty-items and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/kitty-items/landing-page.png<img src>
Linked from: https://developers.flow.com/learn/kitty-items and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/kitty-items/kitty-items-diagram.png<img src>
Linked from: https://developers.flow.com/learn/kitty-items and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image4.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image2.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image3.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image7.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image8.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image1.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image6.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/architectureImages/image5.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/DappArchitectures and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
503 Service unavailable	
http://kitty-items-flow-testnet.herokuapp.com/<a href>Outbound
Linked from: https://developers.flow.com/tools/fcl-js/reference/api and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/kitty-items/blob/master/web/src/hooks/use-current-user.hook.js<a href>Outbound
Linked from: https://developers.flow.com/tools/fcl-js/reference/api and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://docs.onflow.org/access-api/<a href>OutboundRedirected
Linked from: https://developers.flow.com/tools/fcl-js/reference/api and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/flow-port/blob/staging/src/pages/transaction-status.js<a href>Outbound
Linked from: https://developers.flow.com/tools/fcl-js/reference/api and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/tools/fcl-js/configure-fcl<a href>
Linked from: https://developers.flow.com/tools/fcl-js/reference/scripts and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/tools/fcl-js/images/wc-discovery.png<img src>
Linked from: https://developers.flow.com/tools/fcl-js/reference/wallet-connect
SSL handshake error	
https://www.versus-flow.art/<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/user-accounts-and-wallets
404 Not found	
https://github.com/onflow/kitty-items/blob/master/api/src/workers/sale-offer-handler.ts<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure
404 Not found	
https://github.com/onflow/kitty-items/blob/master/web/src/flow/script.get-market-item.js<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure
404 Not found	
https://github.com/onflow/kitty-items/blob/master/api/src/services/market.ts<a href>Outbound
Linked from: https://developers.flow.com/flow/dapp-development/dapp-infrastructure
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/NFTimages/image1.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/NFTimages/image2.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/NFTimages/image3.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/NFTimages/image4.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/NFTimages/image5.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/NFTimages/image6.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/NFT-drop-styles
404 Not found	
https://developers.flow.com/port-login.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/select-wallet.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/port-dashboard.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/port-new-account.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/port-approve.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/port-event.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/flow/dapp-development/port-send.png<img src>
Linked from: https://developers.flow.com/flow/dapp-development/mainnet-account-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/flowscan-fees.png<img src>
Linked from: https://developers.flow.com/learn/concepts/variable-transaction-fees and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://sandboxnet-faucet.onflow.org/<a href>OutboundNew
Linked from: https://developers.flow.com/learn/concepts/accessing-sandboxnet
500 Internal server error	
https://developers.flow.com/assets/learn/concepts/port-sealed-tx.png<img src>New
Linked from: https://developers.flow.com/learn/concepts/accessing-mainnet
404 Not found	
https://developers.flow.com/flow-token/delivery<a href>
Linked from: https://developers.flow.com/flow/flow-token and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://developers.flow.com/flow-token/earn<a href>
Linked from: https://developers.flow.com/flow/flow-token and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/nft-marketplace/building-blocks<a href>
Linked from: https://developers.flow.com/flow/nft-marketplace and [6 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/nft-marketplace/best-practices<a href>
Linked from: https://developers.flow.com/flow/nft-marketplace and [6 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/nft-marketplace/handling-accounts<a href>
Linked from: https://developers.flow.com/flow/nft-marketplace and [6 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/nft-marketplace/minting-nfts<a href>
Linked from: https://developers.flow.com/flow/nft-marketplace and [6 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/nft-marketplace/selling-nfts<a href>
Linked from: https://developers.flow.com/flow/nft-marketplace and [6 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
http://developers.flow.com/flow/fusdSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/fusd and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/flow/fusd/<a href>Redirected
Linked from: https://developers.flow.com/flow/fusd and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/open-source-template<a href>Outbound
Linked from: https://developers.flow.com/cadence/style-guide/project-development-tips and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
http://developers.flow.com/mobile/design-patternsSocial meta tagRedirectedNew
Linked from: https://developers.flow.com/mobile/design-patterns
404 Not found	
http://developers.flow.com/mobile/contract-upgradesSocial meta tagRedirectedNew
Linked from: https://developers.flow.com/mobile/contract-upgrades
404 Not found	
http://developers.flow.com/flow/access-apiSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/access-api
Connect error	
http://localhost:3001/<a href>Outbound
Linked from: https://developers.flow.com/learn/kitty-items/start and [2 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/kitty-items/welcome-ui.png<img src>
Linked from: https://developers.flow.com/learn/kitty-items/start and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/kitty-items/admin-login.png<img src>
Linked from: https://developers.flow.com/learn/kitty-items/start and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
Connect error	
http://localhost:3001/admin/mint/<a href>Outbound
Linked from: https://developers.flow.com/learn/kitty-items/start and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/kitty-items/admin-ui.png<img src>
Linked from: https://developers.flow.com/learn/kitty-items/start and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/learn/kitty-items/shades-nft.png<img src>
Linked from: https://developers.flow.com/learn/kitty-items/modify and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/nodes/node-operation/flow-architecture.png<img src>
Linked from: https://developers.flow.com/nodes/node-operation/node-setup and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/nodes/staking/epoch-service-event-diagram.png<img src>
Linked from: https://developers.flow.com/nodes/staking/epoch-scripts-events and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/flow-core-contracts/blob/master/transactions/lockedTokens/admin/create_admin_collection.cdc<a href>Outbound
Linked from: https://developers.flow.com/flow/core-contracts/locked-tokens and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://github.com/onflow/flow/blob/master/docs/content/access-api.md<a href>Outbound
Linked from: https://developers.flow.com/nodes/node-operation/observer-node
500 Internal server error	
https://developers.flow.com/assets/nodes/node-operation/observer.png<img src>
Linked from: https://developers.flow.com/nodes/node-operation/observer-node
404 Not found	
https://developers.flow.com/concepts/transaction-lifecycle<a href>
Linked from: https://developers.flow.com/nodes/access-api
500 Internal server error	
https://developers.flow.com/assets/cadence/language/type-hierarchy.png<img src>
Linked from: https://developers.flow.com/cadence/language/type-hierarchy
500 Internal server error	
https://developers.flow.com/assets/flow/flow-token/dapper-web.png<img src>
Linked from: https://developers.flow.com/flow/flow-token/available-wallets
500 Internal server error	
https://developers.flow.com/assets/flow/flow-token/dapper-login.png<img src>
Linked from: https://developers.flow.com/flow/flow-token/available-wallets
404 Not found	
http://developers.flow.com/tools/fcl-js/configure-fclSocial meta tagRedirected
Linked from: https://developers.flow.com/tools/fcl-js/configure-fcl
500 Internal server error	
https://developers.flow.com/fusd/<a href>Redirected
Linked from: https://developers.flow.com/flow/fusd/providers
404 Not found	
https://support.ledger.com/hc/en-us/articles/360006395553<a href>OutboundNoindex
Linked from: https://developers.flow.com/nodes/flow-port and [3 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
http://developers.flow.com/flow/nft-marketplace/building-blocksSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/nft-marketplace/building-blocks
500 Internal server error	
http://developers.flow.com/flow/nft-marketplace/best-practicesSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/nft-marketplace/best-practices
500 Internal server error	
http://developers.flow.com/flow/nft-marketplace/handling-accountsSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/nft-marketplace/handling-accounts
500 Internal server error	
http://developers.flow.com/flow/nft-marketplace/minting-nftsSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/nft-marketplace/minting-nfts
500 Internal server error	
http://developers.flow.com/flow/nft-marketplace/selling-nftsSocial meta tagRedirected
Linked from: https://developers.flow.com/flow/nft-marketplace/selling-nfts
404 Not found	
https://access-testnet.onflow.org/<a href>Outbound
Linked from: https://developers.flow.com/flow/faq/operators
404 Not found	
https://access-sandboxnet.onflow.org/<a href>Outbound
Linked from: https://developers.flow.com/flow/faq/operators
404 Not found	
https://access.onflow.org/<a href>Outbound
Linked from: https://developers.flow.com/flow/faq/operators
500 Internal server error	
https://developers.flow.com/nodes/(https://developers.flow.com/nodes/access-api-rate-limits)<a href>New
Linked from: https://developers.flow.com/nodes/archive-api-rate-limits
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-0-00.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
404 Not found	
https://docs.onflow.org/concepts/nodes/node-operation/quickstart/<a href>OutboundRedirected
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-0-02.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100#)
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-0-03.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-0-04.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-0-05.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-4.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/staking-collection.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/machine-account.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-delegate-1.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-delegate-2.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-delegate-3.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-delegate-4.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-1.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-2.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-3.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-5.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/flow-port/port-stake-6.png<img src>Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and 1 more |  
500 Internal server error | https://developers.flow.com/assets/nodes/staking/epoch-phase-diagram.png<img src>Linked from: https://developers.flow.com/nodes/staking/schedule |  
500 Internal server error | http://developers.flow.com/nodes/(https://developers.flow.com/nodes/access-api-rate-limits)Social meta tagRedirectedNewLinked from: https://developers.flow.com/nodes/(https://developers.flow.com/nodes/access-api-rate-limits)
https://developers.flow.com/assets/nodes/flow-port/port-stake-0-03.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-0-04.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-0-05.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-4.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/staking-collection.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/machine-account.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-delegate-1.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-delegate-2.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-delegate-3.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-delegate-4.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-1.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-2.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-3.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-5.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/flow-port/port-stake-6.png<img src>
Linked from: https://developers.flow.com/nodes/flow-port/staking-guide and [1 more](https://www.drlinkcheck.com/account/subscriptions/1/projects/1/reports/broken?count=100&offset=1807#)
500 Internal server error	
https://developers.flow.com/assets/nodes/staking/epoch-phase-diagram.png<img src>
Linked from: https://developers.flow.com/nodes/staking/schedule
500 Internal server error	
http://developers.flow.com/nodes/(https://developers.flow.com/nodes/access-api-rate-limits)Social meta tagRedirectedNew
Linked from: https://developers.flow.com/nodes/(https://developers.flow.com/nodes/access-api-rate-limits)
```

**Comment by bthaile** [comment]
broken links have been significantly reduced, going to open up separate issues per broken link.
